pallets/baby-liminal/src/benchmarking/suite.rs

```diff
@@ -156,5 +156,19 @@ benchmarks! {
         )
     }
 
+    // Cryptography
+
+    poseidon_one_to_one {} : {
```

Comment:
shouldn't the benchmarks be over a range of values? You could easily plug a range of integers there

Reply:
yes, but hashing function should run roughly the same time for different outputs; but I can add an artificial input anyway

Reply:
done